company-lsp.el

```diff
@@ -230,7 +234,6 @@ CANDIDATE is a string returned by `company-lsp--make-candidate'."
          (item (company-lsp--candidate-item resolved-candidate))
          (prefix (company-lsp--candidate-prefix candidate))
```

Comment:
Can you rename `prefix` to `prefix-line`?

Also rename `company-lsp--candidate-prefix` to `company-lsp--candidate-prefix-line` and the symbol it's using for text properties. `company-lsp--make-candidate` should be updated accordingly as well.

Reply:
Fixed.